dotCMS/src/main/java/com/dotcms/auth/providers/saml/v1/DotSamlResource.java

```diff
@@ -0,0 +1,232 @@
+package com.dotcms.auth.providers.saml.v1;
+
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.saml.Attributes;
+import com.dotcms.saml.DotSamlConstants;
+import com.dotcms.saml.DotSamlProxyFactory;
+import com.dotcms.saml.IdentityProviderConfiguration;
+import com.dotcms.saml.IdentityProviderConfigurationFactory;
+import com.dotcms.saml.SamlAuthenticationService;
+import com.dotcms.saml.SamlException;
```

Comment:
rename to DotSamlException for consistency

Reply:
Done